resources/worldconfig.ini

```diff
@@ -77,3 +77,6 @@ allow_players_to_skip_cinematics=0
 # Customizable message for what to say when there is a cdclient fdb mismatch
 cdclient_mismatch_title=Version out of date
 cdclient_mismatch_message=We detected that your client is out of date. Please update your client to the latest version.
+
+# Auto reject properties which contain no models | must be 1 in order to auto reject.
+auto_reject_empty_properties=0
```

Comment:
Imo, default this to 1

Reply:
I set this to 0 to mimic live behavior since if the server communications to the client were implemented properly, the client would auto-display the property as approved and not bother displaying a moderation status whether or not it would have had one.